apps/site/assets/css/_alerts.scss

```diff
@@ -374,3 +374,18 @@
     top: calc(#{$base-spacing} / 1.75);
   }
 }
+
+.c-service-changes-container {
+  display: flex; 
+  color:black;
+}
+
+.c-service-changes-content {
+  width: 32px;
+  height: 24px;
```

Comment:
I think we can probably get away with not hardcoding these using some `em`-based padding, something like this maybe?
```
padding: <y>em <x>em;
```